pages/about.py

```diff
@@ -0,0 +1,118 @@
+                        style={'background-color': 'white', 'padding': '1rem 2rem', 'border-radius': '10px', 'width': '590px', 'min-height': '150px', 'text-align': 'center', 'margin-right': '20px'},
+                        children=[
+                            html.H2('Data Collection', style={'color': '#19264f'}),
+                            html.P('Our data collection process involved gathering traffic volume data from Ontario\'s Ministry of Transportation, analyzing charging station utilization across the province, and referencing successful charging network projects in the United States, such as the West Coast Pacific Highway initiative. By synthesizing insights from these diverse sources, we gained valuable information to inform our strategic planning efforts for optimizing EV charging infrastructure in Ontario.', style={'color': '#4C4C4C'})
+                        ]
+                    )
+                ),
+                html.Br(),
+                html.H3('Event Simulation', id='simulation-flow', style={'font-weight': 'bold'}),
+                html.Div('The simulation primarily tracks the series of events that will occur throughout the process of charging a vehicle. Critical parameters such as driving speed, balking probability due to long queues, and '
+                         'duration of charging stall service time were assumed based on historical data. Future steps involve large-scale SimPy execution in Python'),
+                html.Br(),
+                html.Div(
+                    style={'display': 'flex', 'justify-content': 'center', 'align-items': 'center'},
+                    children=html.Img(
+                        src='assets/simulation-flow_chart.png',
+                        style={'max-height': '250px', 'align-items': 'center'}
+                    )
+                ),
+                html.Br(),
+                html.H3('System Flow Diagram', id='system-flow', style={'font-weight': 'bold'}),
+                html.Div('The following diagram illustrates the relationship between the tools, and the FAQ section below could address some of the questions you might have. For each model, certain features will be necessary. For all features, if future projections cannot be obtained then they will have to be forecasted.'),
+                html.Div(
+                    style={'display': 'flex', 'justify-content': 'center', 'align-items': 'center'},
+                    children=html.Img(src='assets/final_system.png', style={'align-items': 'center'}),
+                ),
+                html.Br(),
+                html.Br(),
+                html.H3('Frequently Asked Questions', id='faq', style={'font-weight': 'bold'}),
+                html.Div([
+                    html.H5('1. What is Integer Programming (IP)?'),
+                    html.P('IP is a mathematical optimization technique where decision variables are restricted to be integers. Link to our IP problem can be found here:'),
```

Comment:
```suggestion
                    html.P('An integer programming model allows us to define locations of stations as integer decision variables, and define constraints and parameters such that a solver can then find the optimal solution to the defined problem'),
```
Accuracy and Clarity